Bottom: a00dc19722d44a4fb6a7025c7aaceae5bad0b848
Top:    edc240362e0e3ae6b38370bd4c928ca54fc0db31
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 18:41:58 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 3468b9b..399b614 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
